client/signup/steps/woocommerce-install/hooks/use-woop-handling/index.ts

```diff
@@ -54,13 +56,29 @@ export default function useEligibility( siteId: number ): EligibilityHook {
 
 		dispatch( requestEligibility( siteId ) );
 		dispatch( requestProductsList() );
+		dispatch( requestLatestAtomicTransfer( siteId ) );
```

Comment:
I don't think we should be checking this in here. :thinking:

Reply:
Let me bring part of what we talked about here. The idea is to be able to anticipate a blocker error in the confirm step/landing page. For instance, when the site is stuck, we can block the transfer step in advance, either in the confirm step as well as in the landing page.

Reply:
Was not thinking about is_stuck. Good call.

Reply:
me neither. Props to @pablinos

Reply:
I mean about `is_stuck` property :-)